.github/workflows/instrumented-tests.sh

```
@@ -0,0 +1,35 @@
+echo "Installing platform tools..."
+$ANDROID_HOME/cmdline-tools/latest/bin/android sdk install "build-tools;37.0.0" platform-tools "platforms;android-$API_LEVEL"
+echo "Installing system image..."
+$ANDROID_HOME/cmdline-tools/latest/bin/android sdk install $PACKAGE
+echo "Creating emulator..."
+echo no | $ANDROID_HOME/cmdline-tools/latest/bin/avdmanager create avd --force -n test --package $PACKAGE
+echo "Starting emulator..."
+$ANDROID_HOME/emulator/emulator -avd test -no-snapshot-save -no-window -no-metrics -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none -skin $SKIN &
+
+start_time=$(date +%s)
+end_time=$((start_time + 60))
+while [ -z $($ANDROID_HOME/platform-tools/adb shell getprop sys.boot_completed > /dev/null) ]; do
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)